client/lv_utils.py

```diff
@@ -44,7 +44,8 @@
 
 @error.context_aware
 def vg_ramdisk(vg_name, ramdisk_vg_size,
-               ramdisk_basedir, ramdisk_sparse_filename):
+               ramdisk_basedir, ramdisk_sparse_filename,
+               use_ram=True):
```

Comment:
The parameter name could be a little more descriptive. Instead of `use_ram`, what about `use_tmpfs`, or `use_ramdisk`? I guess `use_tmpfs` is better because it makes less assumptions about the particular implementation of tmpfs.